test/test_poolmanager.py

```diff
@@ -322,13 +329,13 @@ def test_override_pool_kwargs_url(self):
             "http://example.com/", pool_kwargs=pool_kwargs
         )
 
-        assert retry.Retry.DEFAULT == default_pool.retries
+        assert retry.Retry.DEFAULT == default_pool.retries  # type: ignore[attr-defined]
```

Comment:
Why does mypy not know about this attribute?

Reply:
Here it's defined in `util.retry` for backwards compatibility like this:
https://github.com/urllib3/urllib3/blob/d8eabf304190c62766e05c79ef44be7ff94cfbcb/src/urllib3/util/retry.py#L511
and we ignored type check for it there.

Reply:
Ah got it. It seems like maybe something we can drop for 2.0? The comment says "For backwards compatibility (equivalent to pre-v1.9)" which was a long time ago...

If not, I think we can still add a type annotation for it like

```py
class Retry:
    # Backward compatibility; assigned outside of the class.
    DEFAULT: "Retry"
```

but the ignore is also fine.

Reply:
@sethmlarson what do you think? can we drop it?

Reply:
I think we should keep `Retry.DEFAULT` and add the type annotation+comment that @bluetech suggested. Thanks for discussing this!

Reply:
Thanks, @bluetech for your suggestion. it removed 5 type ignore from our codebase.